src/components/CoinsTable.jsx

```diff
@@ -0,0 +1,188 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Container, makeStyles, createTheme, TextField, Paper, Table, TableBody, TableHead, TableRow, TableCell, TableContainer, LinearProgress, ThemeProvider, Typography } from '@material-ui/core';
+import { useNavigate } from 'react-router-dom';
+import Pagination from '@material-ui/lab/Pagination';
+import { CoinList } from '../config/api';
+import { CryptoState } from '../CryptoContext';
+import { numberWithCommas } from '../pages/Banner/Carousel';
+
+const useStyles = makeStyles({
```

Comment:
A small critique. As this file is getting larger you might want to abstract the styling into it's own file or folder and just import it. You can also to do that for your get request. Finally,  getcoins should be camelCase: getCoins. Similar to how you named handleSearch.

Reply:
perfect I will adjust some of the  changes..am using Material UI for styling that's why the file is getting a little bit bigger . will take care of the camel case on the next  PR!!